docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.8'
```

Comment:
You can remove this configuration as it's now [obsolete](https://github.com/mailcow/mailcow-dockerized/issues/5797#issuecomment-2010649543).

Reply:
The joys of using ChatGPT! 😂